primitives/src/v1.rs

```diff
@@ -22,7 +22,7 @@ use parity_scale_codec::{Encode, Decode};
 use bitvec::vec::BitVec;
 
 use primitives::RuntimeDebug;
-use runtime_primitives::traits::AppVerify;
+use runtime_primitives::traits::{AppVerify, Header as HeaderT};
```

Comment:
naming nit: from what I've seen, the `T` suffix is what we use for generics

Reply:
We do this everywhere in Substrate/Polkadot that way.